CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+# Contributing to LinuxBoot/Heads
+
+Thank you for your interest in contributing to LinuxBoot/Heads! We welcome contributions
+from everyone. By participating in this project, you agree to abide by our
+[Code of Conduct](CODE_OF_CONDUCT.md).
+
+## Communication Channels
+
+For the most up-to-date information on our community channels, please visit:
+https://osresearch.net/community/
+
+## Best Practices for Issues and Contributions
+
+Before opening an issue or submitting a contribution, please:
+
+1. Search existing resources (GitHub issues, wiki, Matrix channel history).
```

Comment:
Searching within https://github.com/linuxboot organization could be easier in case of Github, it should cover more ground in one go (issues, wiki, discussions, PRs).